api/analysis.go

```diff
@@ -282,6 +366,16 @@ func (h AnalysisHandler) AppCreate(ctx *gin.Context) {
 		_ = ctx.Error(err)
 		return
 	}
+	analysis := r.Model()
+	analysis.ApplicationID = id
+	analysis.CreateUser = h.BaseHandler.CurrentUser(ctx)
+	db := h.DB(ctx)
+	db.Logger = db.Logger.LogMode(logger.Error)
+	err = db.Create(analysis).Error
+	if err != nil {
+		_ = ctx.Error(err)
+		return
+	}
```

Comment:
hint: Moved this code blocker here to use r.Model() to properly transfer analysis fields in the posted resource (like: Commit).